client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -3,34 +3,302 @@
 /**
  * External dependencies
  */
+import get from 'lodash/get';
+import { __ } from '@wordpress/i18n';
 import { Component, Fragment } from '@wordpress/element';
-import { withInstanceId } from '@wordpress/compose';
+import { compose, withInstanceId } from '@wordpress/compose';
+import { Panel, PanelBody, PanelRow } from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { SIMPLE_PAYMENTS_PRODUCT_POST_TYPE } from 'lib/simple-payments/constants';
 
 class Edit extends Component {
-	handleChange = event => {
-		const paymentId = parseInt( event.target.value, 10 );
-		if ( paymentId ) {
-			this.props.setAttributes( { paymentId } );
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			savingProduct: false,
+		};
+	}
+
+	componentDidUpdate( prevProps ) {
+		const { simplePayment, attributes, setAttributes, isSelected, isSaving } = this.props;
+
+		if ( ! prevProps.simplePayment && simplePayment ) {
+			setAttributes( {
+				description: get( simplePayment, 'content.raw', attributes.description ),
```

Comment:
We could save `_.get()` some cycles if we use arrays instead of strings in all these calls, e.g. `[ 'content', 'raw' ]` instead of `'content.raw'`